scripts/new_upstream_snapshot.py

```diff
@@ -363,53 +378,59 @@ def update_changelog(
     # Determine new changelog version
     changelog_version = ""
     if commitish_is_upstream_tag:
-        if first_sru:
-            series_number = capture("distro-info --stable -r").stdout.strip()
-            changelog_version = f"{commitish}-0ubuntu0~{series_number}.1"
-        elif is_devel:
+        # Upstream tag always takes priority. For 22.1:
```

Comment:
@blackboxsw , I removed a section you added that bumps the version if quilt patches have been refreshed. I'm not convinced that was actually valid and was instead a workaround to a condition I had missed previously. Let me know if you think I'm missing something by removing it.

Reply:
I'm good with dropping this logic. We don't frequently find ourselves in the scenario where we have only refreshed quilt patches and not included and upstream features via a new_upstream_snapshot anyway. Typical(infrequent) condition where we'd need to refresh quilt patches in a downstream branch would be due to a refactor in main making daily recipes unable to build due to inapplicable quilt patch applies. In this scenario, We don't need to actually bump the package version number if we are only refreshing patches because we aren't planning on releasing that content. We can manually tweak the  UNRELEASED version number in d/changelog if we feel the need, but daily PPA recipe builds don't grok d/changelog.